chess.js

```diff
@@ -27,19 +27,50 @@ function isCheck() {
 }
 
 function isValidKingMove(from, to) {
-    return false
+    const positionFrom = getPosition(from)
+    const positionTo = getPosition(to)
```

Comment:
Pošto ti se ovo ponavlja svugde, možda bi bilo bolje da to prosledimo funkcijama umesto from i to da izbegnemo ponavljanje.

Reply:
Da budu parametri?

Reply:
Da. Umesto da proslediš from i to, odmah prosliediš positionFrom i positionTo.

Reply:
Iskreno nemam pojma kako ovo da odradim. Probacu ponovo malo kasnije